.github/workflows/test-integrations-aws.yml

```diff
@@ -12,8 +12,7 @@ on:
   # XXX: We are using `pull_request_target` instead of `pull_request` because we want
   # this to run on forks with access to the secrets necessary to run the test suite.
   # Prefer to use `pull_request` when possible.
-  pull_request_target:
-    types: [labeled, opened, reopened, synchronize]
+  pull_request:
```

Comment:
This fixed the fact that now the AWS tests are running from the correct branch (the ones you want to merge it)

But this breaks the CI check for a correct test setup. This is fixed in the related PR that only deals with updating the CI setup for AWS: https://github.com/getsentry/sentry-python/pull/4076

Reply:
If we merge #4076 into this branch, all CI will be green.